docs/reference/query-dsl/geo-distance-query.asciidoc

```diff
@@ -15,13 +40,18 @@ distance from a geo point. Assuming the following indexed json:
     }
 }
 --------------------------------------------------
+// CONSOLE
+// TEST[continued]
```

Comment:
I'll follow your suggestion. Wrt. "way too early" - essentially that's one of the reasons I put this PR out early, so we can look at things early as we progress and figure out what we like best.